### DISCUSSION TITLE
restream rtsp

### DISCUSSION THREAD

**Discussion by shah1git** [bluenviron/mediamtx#2039] [Questions]
Hello.

Could you tell me how to set up mediamtx for such a task.

I am getting a rtsp stream from a video camera from a weak channel to my server where mediamtx is running. I want my clients to receive this rtsp stream not from the video camera, but from my server, how to set it up?

**Comment by aler9** [comment]
Hello, you can ingest the video camera feed into the server by using this configuration:

```yml
paths:
  mycamera:
    source: rtsp://original-rtsp-url
```

then you can read the feed from the server by using the URL:

```
rtsp://server-ip:8554/mycamera
```

[a comment by an automated account is not shown]
